engines/registration_sharing/app/controllers/registration_sharing/rmt_to_rmt_controller.rb

```diff
@@ -21,10 +21,7 @@ def create
           )
         end
 
-        system.hw_info ||= HwInfo.create
-        system.hw_info.instance_data = params[:instance_data]
-        system.hw_info.save!
-
+        system.instance_data = params[:instance_data]
```

Comment:
could we remove `instance_data` from `hw_infos` table then ?
[here](https://github.com/SUSE/rmt/blob/08e9887e0deb07d897f10819548b40a93d5b902e/db/schema.rb#L52)